homework_8/geometric_figure/Square.java

```diff
@@ -0,0 +1,22 @@
+package geometric_figure;
+
+public class Square implements GeometricFigure {
+    private double a;
```

Comment:
please never set such names to your variables. It's not clear what is it and the name itself is not informative. Rename it to something more meaningful - `height`, `wight`, or whatever it is.